tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -390,8 +397,47 @@ public void SortHandler()
         }
     }
 
+    private void AddNewlyCreatedComputeSystem(ComputeSystemViewModel computeSystemViewModel)

```

Comment:
Does this assume that computeSystemViewModel is in a certain state?

Reply:
No it does not.The only thing we know is that its been created